observability/opentelemetry-metrics-backend/src/main/scala/sttp/client4/opentelemetry/OpenTelemetryMetricsBackend.scala

```diff
@@ -88,13 +93,13 @@ private object OpenTelemetryMetricsListener {
 private class OpenTelemetryMetricsListener(
     meter: Meter,
     clock: Clock,
-    requestToLatencyHistogramMapper: GenericRequest[_, _] => Option[CollectorConfig],
+    requestToLatencyHistogramMapper: GenericRequest[_, _] => Option[HistogramCollectorConfig],
```

Comment:
Sorry, I'm late for the party.
To generate opentelemetry http metrics is better to use opentelemetry instrumentation api and supported metrics will be generated out of the box